src/main/resources/assets/iris/lang/en_gb.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If you're not changing anything from the en_us file, then no need to have this to begin with.